apps/wing-console/console/ui/src/layout/use-layout.tsx

```diff
@@ -105,6 +106,7 @@ export const useLayout = ({
     },
     {
       enabled: selectedItems.length > 0,
+      keepPreviousData: false,
```

Comment:
Why this change?

Reply:
I added this to prevent the UI from displaying the metadata of the previously selected node when switching from one resource to another. `useLayout` is providing the trpc object directly, and we are using the `metadata.data` in the UI, so we need to clean it up.

We could do other things like check if it's loading I guess..

with:
https://github.com/winglang/wing/assets/5547636/f2d5334e-8e66-4574-8029-352e3ec356db

without:
https://github.com/winglang/wing/assets/5547636/1b69808a-116b-4e75-87f8-08cf3205dc6a

Reply:
Makes sense what you're doing, although I think there's something strange. Even with preserve enabled, there's a flash where the panel is empty 🤔 

Also, when did it get so slow?

Reply:
It's 'slow' as usual, but now the metadata pane was hidden while fetching new data. 
I fixed this by keeping the previos metadata panel until the new metadata is fetched.